src/Slider.jsx

```diff
@@ -694,76 +728,142 @@ class Rheostat extends React.Component {
+      styles,
     } = this.props;
-    const { className, handlePos, values } = this.state;
+
+    const {
+      handleDimensions,
+      handlePos,
+      values,
+    } = this.state;
+
+    const handleContainerStyle = orientation === VERTICAL
+      ? { left: 0, bottom: handleDimensions / 2, top: handleDimensions / 2 }
+      : { top: 0, left: handleDimensions / 2, right: handleDimensions / 2 };
 
     return (
       // eslint-disable-next-line jsx-a11y/click-events-have-key-events
       <div
-        className={className}
-        ref={this.setRef}
-        onClick={!disabled ? this.handleClick : undefined}
-        style={{ position: 'relative' }}
+        onClick={!disabled && this.handleClick}
+        {...css(
+          styles.rheostat,
+          autoAdjustVerticalPosition && styles.autoAdjustVerticalPosition,
+          orientation === VERTICAL && styles.rheostat_vertical,
+        )}
       >
-        <div className="rheostat-background" />
-        {handlePos.map((pos, idx) => {
-          const handleStyle = orientation === 'vertical'
-            ? { top: `${pos}%`, position: 'absolute' }
-            : { left: `${pos}%`, position: 'absolute' };
+        <div
+          ref={this.setHandleContainerNode}
+          {...css(
+            styles.handleContainer,
+            handleContainerStyle,
+            styles.Rheostat_background,
+            orientation === VERTICAL
+            ? styles.Rheostat_background__vertical
+            : styles.Rheostat_background__horizontal,
+          )}
+        >
+          {handlePos.map((pos, idx) => {
+            const handleStyle = orientation === VERTICAL
+              ? { top: `${pos}%`, position: 'absolute' }
+              : { left: `${pos}%`, position: 'absolute' };
 
-          return (
-            <Handle
-              aria-valuemax={this.getMaxValue(idx)}
-              aria-valuemin={this.getMinValue(idx)}
-              aria-valuenow={values[idx]}
-              aria-disabled={disabled}
-              data-handle-key={idx}
-              className="rheostat-handle"
-              key={`handle-${idx}`}
-              onClick={this.killEvent}
-              onKeyDown={!disabled ? this.handleKeydown : undefined}
-              onMouseDown={!disabled ? this.startMouseSlide : undefined}
-              onTouchStart={!disabled ? this.startTouchSlide : undefined}
-              role="slider"
-              style={handleStyle}
-              tabIndex={0}
-            />
-          );
-        })}
-        {handlePos.map((node, idx, arr) => {
+            return (
+              <Handle
+                aria-valuemax={this.getMaxValue(idx)}
+                aria-valuemin={this.getMinValue(idx)}
+                aria-valuenow={values[idx]}
+                aria-disabled={disabled}
+                data-handle-key={idx}
+                key={
+                  // eslint-disable-next-line react/no-array-index-key
+                  idx
+                }
```

Comment:
you could probs have this be:
```
key={idx} // eslint-disable-line react/no-array-index-key
```

Reply:
I don't think I can do this (this jsx doesn't allow comments outside of `{ }` blocks, but:

`key={idx /* eslint-disable-line react/no-array-index-key */}` worked!